testdata/baselines/reference/submoduleAccepted/conformance/checkExportsObjectAssignProperty.errors.txt.diff

```diff
@@ -0,0 +1,147 @@
+--- old.checkExportsObjectAssignProperty.errors.txt
++++ new.checkExportsObjectAssignProperty.errors.txt
+@@= skipped -0, +0 lines =@@
+-validator.ts(17,4): error TS2540: Cannot assign to 'readonlyProp' because it is a read-only property.
+-validator.ts(18,4): error TS2540: Cannot assign to 'readonlyAccessor' because it is a read-only property.
+-validator.ts(19,1): error TS2322: Type 'string' is not assignable to type 'number'.
+-validator.ts(20,1): error TS2322: Type 'string' is not assignable to type 'number'.
+-validator.ts(21,1): error TS2322: Type 'number' is not assignable to type 'string'.
+-validator.ts(37,4): error TS2540: Cannot assign to 'readonlyProp' because it is a read-only property.
+-validator.ts(38,4): error TS2540: Cannot assign to 'readonlyAccessor' because it is a read-only property.
+-validator.ts(39,1): error TS2322: Type 'number' is not assignable to type 'string'.
+-validator.ts(40,1): error TS2322: Type 'string' is not assignable to type 'number'.
+-validator.ts(41,1): error TS2322: Type 'number' is not assignable to type 'string'.
+-
+-
+-==== validator.ts (10 errors) ====
++index.js(4,11): error TS2580: Cannot find name 'require'. Do you need to install type definitions for node? Try `npm i --save-dev @types/node`.
```

Comment:
`mod1` looks like:

```
Object.defineProperty(exports, "thing", { value: 42, writable: true });
Object.defineProperty(exports, "readonlyProp", { value: "Smith", writable: false });
Object.defineProperty(exports, "rwAccessors", { get() { return 98122 }, set(_) { /*ignore*/ } });
Object.defineProperty(exports, "readonlyAccessor", { get() { return 21.75 } });
Object.defineProperty(exports, "setonlyAccessor", {
    /** @param {string} str */
    set(str) {
        this.rwAccessors = Number(str) 
    }
});
```

This is not being treated as a module for some reason? I don't think I did anything to affect this, probably another bug.